packages/graphback-cli/src/templates/resources/docker/pg-docker-compose.yml

```diff
@@ -10,10 +10,7 @@ services:
       POSTGRES_USER: postgresql
       POSTGRES_DB: users
 
-  # Mosca is a simple MQTT Broker
-  # In OpenShift/Production we would use the Red Hat AMQ broker
-  mosca:
-    image: matteocollina/mosca
+  redis:
```

Comment:
pg docker compose has redis? We are ok with that but we might create issue to abstract it

Reply:
Redis will be common, as it is for subscriptions. pg docker compose specifies database as postgres

Reply:
ok